src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,41 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public static final int FIGURE_COUNT = 5;
+    public static final int MAX_RANDOM_NUMBER = 20;
+    public static final double DEFAULT_RADIUS = 10.0;
+
+    public Figure getRandomFigure() {
+        Random random = new Random();
+        int figureNumber = random.nextInt(FIGURE_COUNT);
+        Color randomColor = new ColorSupplier().getRandomColor();
+
+        switch (figureNumber) {
+            case 1:
+                double radius = random.nextDouble() * MAX_RANDOM_NUMBER;
+                return new Circle(radius, randomColor);
+            case 2:
```

Comment:
let's create separate methods for each of figure and move such logic there. For example for circle method getRandomCircle()

Reply:
if i create separate methods for each figure, I have to create this variable in each methods too?
Random random = new Random();
int figureNumber = random.nextInt(FIGURE_COUNT);
Color randomColor = new ColorSupplier().getRandomColor();